apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -351,6 +368,9 @@ k8s-web:
 		KUMA_CPU_REQUEST=${WEB_CPU_REQUEST} \
 		KUMA_MEMORY_REQUEST=${WEB_MEMORY_REQUEST} \
 		KUMA_ALLOWED_HOSTS=${WEB_ALLOWED_HOSTS} \
+		NEW_RELIC_APP_NAME=${NEW_RELIC_WEB_NAME} \
+		NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
+		NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```

Comment:
I think these two lines:
```
NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```
are not needed since they will be picked up from the existing exported env variables?

Reply:
ok, I was just pasting the vars into each section to make it look like the others, but it appears these are for overrides only. fixed each section.